src/components/collaborative-editing/components/avatars/index.js

```diff
@@ -35,13 +36,24 @@ export function CollaborativeEditingAvatars( { peers } ) {
 
 export function CollaborativeEditingAvatar( { peer } ) {
 	const [ isVisible, setIsVisible ] = useState( false );
+	const { selectBlock } = useDispatch( blockEditorStore );
```

Comment:
We generally pass in string identifiers (`'core/block-editor'`) instead of the store object.

Reply:
What do you think about moving the dispatch call into the connected component, like we do with the `select`? We could pass down an onClick function prop to this component.

Keeping the pure component and connected component separated makes it easier to mock/test/reuse.